amazon-kinesis-client/src/main/java/software/amazon/kinesis/common/StreamConfig.java

```diff
@@ -6,6 +6,7 @@
 @Value
```

Comment:
Since we have shardDetector now mapped against streamConfig in scheduler, I feel this should include the intialPosition as well. Removing the comment.